### ISSUE TITLE
[MAS4.2.5] [Labels and Relationships - Settings] Voiceover announce the same label for “Path to ngrok", "Locale", "Localhost Override"and "User ID" present in the "settings" pane.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1903]
**Actual Result:**
In settings pane, the voiceover announces the incorrect labels for edit fields i.e. "Path to ngrok", "Locale", "Localhost Override" and "User ID" whereas the labels are correctly defined in the code.
Voiceover announces "Enter User ID" edit text for all the edit fields.

**Expected Result:**
The voiceover should announce the label which is associated with the edit fields so that user can understand the purpose & provides valid input.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Open Bot" button and activate it.
4. Open a Bot Dialog box gets opened.
5. Navigate to the "bot URL" edit box and enter URL- http://localhost:3978/api/messages
6. Navigate to "Connect" button and activate it.
7. "Live Chat" separate window appears.
8. Navigate to "edit ngrok settings" controls available at right lower corner of the pane and activate it.
9. Observe the issue.

**User Impact:**
It will be difficult for visually impaired user to enter the required input, if the text box label's are not properly announced.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={07857fe1-d8ef-409e-a594-f440a8359566}](url)

**Attachment:**
[Voiceover announce the same label for edit boxes.zip](https://github.com/microsoft/BotFramework-Emulator/files/3689467/Voiceover.announce.the.same.label.for.edit.boxes.zip)

**Comment by ShikhaMishra11** [comment]
Checked and verified. Issue resolved in new version of Bot Emulator. Hence, closing the issue.

Test Environment:
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.6.0-89098)
